src/components/pages/UserNotSetUpPage.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const UserNotSetUpPage = () => {
+
+return (
+        <div>
+            <h1>Your User Has Not Been Fully Set Up</h1>
+            <p>Please raise an issue in the <strong>#dataworks-aws-analytical-env-testing</strong> Slack channel.</p>
```

Comment:
Change title to: Your user is not yet authorized to use this system, or is in the process of being setup.
Change the detail to: Please contact technical support if you believe this information to be incorrect. "Inconvenience" spelt incorrectly, but please remove the statement about being sorry.